src/basho_bench_worker.erl

```diff
@@ -102,12 +106,17 @@ init([SupChild, Id]) ->
     Ops     = ops_tuple(),
     ShutdownOnError = basho_bench_config:get(shutdown_on_error, false),
 
+    %% Check configuration for flag enabling new API that passes opaque State object to support accessor functions
+    ApiPassState = basho_bench_config:get(api_pass_state),
```

Comment:
If this is going to be an optional new flag you'll likely want to add a default `false` value here so that it doesn't crash if `api_pass_state` is not defined in the config.